### PR TITLE
Use CircleCI context instead of environment variables

### PR DESCRIPTION
**Pull request by yuokada**
## Tasks

- [x] Drop the below environment variables in the following page after merging this PR
       https://app.circleci.com/settings/project/github/treasure-data/presto/environment-variables
       
       - AWS_ACCESS_KEY_ID
       - AWS_SECRET_ACCESS_KEY
       - TD_ARTIFACTORY_PASSWORD
       - TD_ARTIFACTORY_USERNAME
       - (extra) MAVEN_OPTS 🆕

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -41,8 +42,15 @@ jobs:
...
 
+      - run:
+          name: Debug
+          command: |
+            echo $MAVEN_OPTS | base64
+
       - run:
           name: Build presto
+          # environment:
+          #   MAVEN_OPTS: "true"
```
Comment (takezoe) [inline .circleci/config.yml]: Debug step above and these comments can be dropped?

Comment (yuokada) [reply .circleci/config.yml]: Of course. Sorry, I forgot to delete.

**Comment by yuokada** [comment]
Confirmed that the `MAVEN_OPTS` env variable in the following page is able to be deleted.
https://app.circleci.com/settings/project/github/treasure-data/presto/environment-variables

https://app.circleci.com/pipelines/github/treasure-data/presto/341/workflows/fe630a78-3487-4fc8-ad70-74568467da8d/jobs/354

```
#!/bin/bash -eo pipefail
echo $MAVEN_OPTS | base64

LVhteDRnIC1YbXM0Zwo=  # => "-Xmx4g -Xms4g". This value is hardcoded in config.yml.
```

File: .circleci/config.yml (hunk 1 above)
Comment (yuokada) [reply .circleci/config.yml]: @takezoe Fixed. PTAL.

**Review by takezoe — APPROVED** [review]
LGTM

**Comment by yuokada** [comment]
Cherry-picked the commit to the `hotfix-350` branch.
https://github.com/treasure-data/presto/commit/7d1948843d560b501074196dd2c9ce6a7e577979
https://app.circleci.com/pipelines/github/treasure-data/presto/345/workflows/ecebe3e4-8bb9-4bd6-8584-c170276d116b

**Pull request merged**